src/sign.builds

```diff
@@ -10,8 +10,26 @@
 
   <UsingTask AssemblyFile="$(BuildToolsTaskDir)Microsoft.DotNet.Build.Tasks.dll" TaskName="ReadSigningRequired" />
 
+  <PropertyGroup Condition="'$(OSGroup)' == 'Windows_NT' and '$(SkipSigning)' != 'true'">
+    <NativePlatform Condition="'$(Platform)' != 'AnyCPU'">$(Platform)</NativePlatform>
+    <NativePlatform Condition="'$(Platform)' == 'AnyCPU'">x64</NativePlatform>
+    <CLRCompressionLocation>$(BinDir)\$(OSGroup).$(NativePlatform).$(ConfigurationGroup)\native\clrcompression.dll</CLRCompressionLocation>
```

Comment:
I would suggest putting these into an itemgroup and batching on it in your target so it is a little easier to extend if needed.